examples/build/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Just saw this. `build/` folders should not be committed to Github. Can you remove this folder, or add a line in .gitignore? Idem for the favicon.